code/modules/clothing/suits/jobs.dm

```diff
@@ -263,6 +282,16 @@ obj/item/clothing/suit/apron/overalls/blue
 	icon_state = "hazard_ce"
 	item_state = "hazard_ce"
 
+/obj/item/clothing/suit/storage/iacvest // todo : make this hazard vest subtype
```

Comment:
why is it not hazard vest subtype right now?

Reply:
there's no zipped icon state at the moment

i'm planning to move all IAC stuff to a contained sprite dmi in the next PR